### PR TITLE
New version: JAMSoftware.TreeSize.Free version 4.6.1

### PR DESCRIPTION
**Pull request by Daniel-Tr**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80215)

### CODE REVIEW

File: manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize.Free
+PackageVersion: 4.6.1
```
Comment (ItzLevvie) [inline manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.installer.yaml:5]:
```suggestion
PackageVersion: 4.6
```
I'm still seeing `4.6` in Control Panel:
image

File: manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize.Free
+PackageVersion: 4.6.1
```
Comment (ItzLevvie) [inline manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 4.6
```
I'm still seeing `4.6` in Control Panel:
image

File: manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-22000-832.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize.Free
+PackageVersion: 4.6.1
```
Comment (ItzLevvie) [inline manifests/j/JAMSoftware/TreeSize/Free/4.6.1/JAMSoftware.TreeSize.Free.yaml:5]:
```suggestion
PackageVersion: 4.6
```
I'm still seeing `4.6` in Control Panel:
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Daniel-Tr** [comment]
@ItzLevvie: Package version 4.6.1 is correct - the product version set is wrong unfortunately (we missed to update it along with the file version, which is set to 4.6.1 as intended). We will publish a corrected version shortly.

**Comment by Daniel-Tr** [comment]
Corrected version has been published -> Revoking this PR, creating a new one with the correct information

**Pull request closed without merging**